src/main/java/nodes/DialogBox.java

```diff
@@ -0,0 +1,46 @@
+package nodes;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+public class DialogBox extends HBox {
+    private Label text;
+    private ImageView displayPicture;
+
```

Comment:
Should there be JavaDoc comment to tell use what the class represents? I noticed the same issue in several other classes too. It would be good to have a JavaDoc comments on the class so that we can know more information about this class.